ws-nextjs-app/pages/[service]/send/[id]/FormField/FieldLabel.tsx

```diff
@@ -9,8 +9,26 @@ export default ({
   id,
   children,
   className,
-}: PropsWithChildren<{ id: InputProps['id']; className?: string }>) => (
-  <Text as="label" className={className} htmlFor={id} css={styles.fieldLabel}>
-    {children}
-  </Text>
-);
+  isCheckbox,
+}: PropsWithChildren<{
+  id: InputProps['id'];
+  className?: string;
+  isCheckbox?: boolean;
+}>) => {
+  return isCheckbox ? (
+    <Text as="label" className={className} htmlFor={id} css={styles.fieldLabel}>
+      {children}
+    </Text>
+  ) : (
+    <div>
+      <Text
+        as="label"
+        className={className}
+        htmlFor={id}
+        css={styles.fieldLabel}
```

Comment:
Is the `<div>` here to make the label a block element? If so, I would remove the ternary here and conditionally add in a CSS class and use `display:block` if it its anything but a checkbox input.

Reply:
this is so the form looks correct when all css is disabled - i think if we use a css class then it wouldnt work as that would be disabled?

Reply:
May be worth checking if you can wrap a `div` around the `label`. Just in case it causes the screenreader to not associate the label with the input anymore. I tried to find some examples of this, but most documentation around forms places the label right beside the input field.

Reply:
actually just had a reread of catharine's notes on the jira, had misread it! we want the divs round the inputs not the labels, have changed this now and this removes the need for a ternary here

Reply:
Cool 👍 Still may be worth checking if this affects screen-readers since we're adding a new parent wrapper.